[DOC-12408]: Feedback on Index Service Metrics | Couchbase Docs

RayOffiah opened the pull request:
Additional details for index cache hits and misses.

Minor fix for the template.

Preview URL:
https://preview.docs-test.couchbase.com/docs-server-DOC-12408/release/8.0/Feedback-on-Index-Service-Metrics-/server/current/metrics-reference/index-service-metrics.html

You will need the [Docs Team credentials](https://confluence.issues.couchbase.com/wiki/spaces/DOCS/pages/1971224949/Docs+Team+demo+site+passwords) on Confluence.

RayOffiah committed:
[DOC-12408]: Feedback on Index Service Metrics | Couchbase Docs

Added additional details for `index_cache_hits` and `index_cache_misses`

Fixed minor error on the template.

Signed-off-by: Ray Offiah <ray.offiah@couchbase.com>

osfameron reviewed (changes requested):
er wait, we shouldn't edit that file directly

osfameron commented:
e.g. the feedback needs to go to the devs, to add to the Index service metrics on their end.
When they build, we can pick up the latest version from the build tarball.

(If we update this file directly, then the next time the devs update the metrics and we update the tarball, we'll just lose it again.)

RayOffiah commented:
Ah, right okay. 

I'll have a chat with the devs. 👍🏾

RayOffiah committed:
[DOC-12408]: Feedback on Index Service Metrics | Couchbase Docs

Changes following review. The change is being made here so we don't forget what it says!

Signed-off-by: Ray Offiah <ray.offiah@couchbase.com>
